modules/ROOT/partials/configuration/toolbar_location.adoc

```diff
@@ -17,6 +18,7 @@ NOTE: `auto` was added and set as the default in {productname} 5.3. For {product
 
 *Possible values:* `auto`, `top`, `bottom`
 
+[discrete]
```

Comment:
I don't think this should be used, no other page uses this discrete attribute. Other pages have the examples show up in the ToC.